Bender.yml

```diff
@@ -106,143 +146,55 @@ sources:
-      - corev_apu/fpga/src/apb_uart/src/slib_input_filter.vhd
-      - corev_apu/fpga/src/apb_uart/src/slib_counter.vhd
-      - corev_apu/fpga/src/apb_uart/src/uart_receiver.vhd
-      - corev_apu/fpga/src/apb_uart/src/slib_input_sync.vhd
-      - corev_apu/fpga/src/apb_uart/src/slib_edge_detect.vhd
-      - corev_apu/fpga/src/apb_uart/src/slib_clock_div.vhd
-      - corev_apu/fpga/src/apb_uart/src/slib_fifo.vhd
-      - corev_apu/fpga/src/apb_uart/src/uart_baudgen.vhd
+
+    # - target: test
+    #   files:
+    #   - corev_apu/tb/ariane_soc_pkg.sv
+    #   - corev_apu/tb/ariane_axi_soc_pkg.sv
+    #   - corev_apu/tb/ariane_testharness.sv
+    #   - corev_apu/tb/ariane_peripherals.sv
+    #   - corev_apu/tb/common/uart.sv
+    #   - corev_apu/tb/common/SimDTM.sv
+    #   - corev_apu/tb/common/SimJTAG.sv
+      
+      
+    # - target: not(synthesis)
+    #   files:
+    #     - common/local/util/instr_tracer.sv
+    #     - common/local/util/instr_tracer_if.sv
+    #     - common/local/util/instr_trace_item.svh
+    #     - common/local/util/ex_trace_item.svh
+    # - target: all(fpga, xilinx)
+    #   files:
+    #   - corev_apu/fpga/src/ariane_peripherals_xilinx.sv
+    #   - corev_apu/fpga/src/ariane_xilinx.sv
+    #   - corev_apu/fpga/src/fan_ctrl.sv
+    #   - corev_apu/fpga/src/bootrom/bootrom.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/ssio_ddr_in.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/rgmii_soc.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/axis_gmii_rx.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/oddr.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/axis_gmii_tx.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/dualmem_widen8.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/rgmii_phy_if.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/dualmem_widen.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/rgmii_lfsr.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/rgmii_core.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/eth_mac_1g.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/eth_mac_1g_rgmii.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/eth_mac_1g_rgmii_fifo.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/iddr.sv
+    #   - corev_apu/fpga/src/ariane-ethernet/framing_top.sv
+    #   - corev_apu/fpga/src/apb_uart/src/apb_uart.vhd
+    #   - corev_apu/fpga/src/apb_uart/src/uart_transmitter.vhd
+    #   - corev_apu/fpga/src/apb_uart/src/uart_interrupt.vhd
+    #   - corev_apu/fpga/src/apb_uart/src/slib_mv_filter.vhd
+    #   - corev_apu/fpga/src/apb_uart/src/slib_input_filter.vhd
+    #   - corev_apu/fpga/src/apb_uart/src/slib_counter.vhd
+    #   - corev_apu/fpga/src/apb_uart/src/uart_receiver.vhd
+    #   - corev_apu/fpga/src/apb_uart/src/slib_input_sync.vhd
+    #   - corev_apu/fpga/src/apb_uart/src/slib_edge_detect.vhd
+    #   - corev_apu/fpga/src/apb_uart/src/slib_clock_div.vhd
+    #   - corev_apu/fpga/src/apb_uart/src/slib_fifo.vhd
+    #   - corev_apu/fpga/src/apb_uart/src/uart_baudgen.vhd
```

Comment:
To make this more general, could we perhaps add a target `ariane_soc` or similar to include the files needed to compile `ariane_testharness` or `ariane_xilinx`?